<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConsolidateUnspentWizardSelectInputsPage</class>
 <widget class="QWizardPage" name="ConsolidateUnspentWizardSelectInputsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="instructionsHorizontalLayout">
       <item>
        <widget class="QLabel" name="selectInputsIntroLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Step 1: Select the inputs to be consolidated. Remember that the inputs to the consolidation are your unspent outputs (UTXOs) in your wallet.</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="treeHorizontalLayout">
       <item>
        <widget class="QPushButton" name="selectAllPushButton">
         <property name="text">
          <string>Select All</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="treeModeRadioButton">
         <property name="text">
          <string>Tree Mode</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="listModeRadioButton">
         <property name="text">
          <string>List Mode</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="treehorizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="filterHorizontalLayout">
       <item>
        <widget class="QLabel" name="filterLabel">
         <property name="text">
          <string>Select inputs</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="filterModePushButton">
         <property name="toolTip">
          <string/>
         </property>
         <property name="text">
          <string>&lt;=</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="BitcoinAmountField" name="maxMinOutputValue">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="filterPushButton">
         <property name="toolTip">
          <string>Filters the already selected inputs.</string>
         </property>
         <property name="text">
          <string>Filter</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="filterHorizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="CoinControlTreeWidget" name="treeWidget">
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="sortingEnabled">
        <bool>false</bool>
       </property>
       <property name="columnCount">
        <number>10</number>
       </property>
       <attribute name="headerShowSortIndicator" stdset="0">
        <bool>true</bool>
       </attribute>
       <attribute name="headerStretchLastSection">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string/>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Amount</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Label</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Address</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Date</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Confirmations</string>
        </property>
        <property name="toolTip">
         <string>Confirmed</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string/>
        </property>
       </column>
       <column>
        <property name="text">
         <string/>
        </property>
       </column>
       <column>
        <property name="text">
         <string/>
        </property>
       </column>
       <column>
        <property name="text">
         <string/>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="summaryHorizontalLayout">
       <item>
        <widget class="QLabel" name="outputLimitWarningIconLabel">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../bitcoin.qrc">:/icons/warning</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="outputLimitStopIconLabel">
         <property name="maximumSize">
          <size>
           <width>64</width>
           <height>64</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../bitcoin.qrc">:/icons/white_and_red_x</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="summaryHorizontalSpacer_1">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="quantityTextLabel">
         <property name="text">
          <string>Quantity</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="quantityLabel">
         <property name="text">
          <string>99999</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="summaryHorizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="feeTextLabel">
         <property name="text">
          <string>Fee</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="feeLabel">
         <property name="text">
          <string>99.9999</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="summaryHorizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="afterFeeTextLabel">
         <property name="text">
          <string>After Fee Amount</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="afterFeeLabel">
         <property name="text">
          <string>999999999.9999</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="isCompleteCheckBox">
         <property name="text">
          <string>isComplete</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CoinControlTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>coincontroltreewidget.h</header>
  </customwidget>
  <customwidget>
   <class>BitcoinAmountField</class>
   <extends>QSpinBox</extends>
   <header>bitcoinamountfield.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../bitcoin.qrc"/>
 </resources>
 <connections/>
</ui>
